UK_Power_Generation_Units_BMRS/Python/Code/ukpowergeneration/ScriptMapQuery/README-BMRS-API-Input-JA-#-ReadAll.md

```diff
@@ -0,0 +1,48 @@
+# README for BMRS-API-Input-JA-#.py ("#" = version)
+## Description
+This document is a README for:
+> BMRS-API-Input-JA-#-ReadAll.py
+
+Three csv files are required. These are not version controlled, but can be found at: 
+> \Dropbox (Cambridge CARES)\CoMo shared\ja685\BMRS\Script-BMRS-API
+Their names are: 
+> Input-Template.csv
+> generator.tripple.csv
+> powerplanttripple.csv
```

Comment:
Please update to:
`dropbox\CoMo shared\ja685\Modelling-Resources\BMRS\Script-BMRS-API`